osdk/README.md

```diff
@@ -76,6 +76,7 @@ Currently we support following commands:
 - **new**: Create a new kernel package or library package
 - **build**: Compile the project and its dependencies
 - **run**: Run the kernel with a VMM
+- **debug**: Debug a remote target via GDB
```

Comment:
Could you also add documentation about the debug support of OSDK in the Asterinas Book? Specifically, `OSDK User Reference > Commands > cargo osdk run` and `OSDK User Reference > Commands > cargo osdk debug`.